addons/spreadsheet/static/src/list/plugins/list_core_plugin.js

```diff
@@ -88,6 +88,7 @@ export class ListCorePlugin extends OdooCorePlugin {
                 }
                 break;
             case "UPDATE_ODOO_LIST_DOMAIN":
+            case "UPDATE_ODOO_LIST_SORTING":
```

Comment:
We had some migration issues with partial updates of datasource, so I would go for a command `UPDATE_ODOO_LIST` which takes a full definition of the list. It's a lot easier to migrate.
In addition, we try to limit the number of corecommand to the minimum, using this strategy helps us.